elements/pfe-jump-links/src/pfe-jump-links.js

```diff
@@ -114,7 +116,7 @@ class PfeJumpLinksNav extends PFElement {
             );
           this.shadowRoot.querySelector(
             "#pfe-jump-links-nav--heading"
-          ).innerHTML = "Jump to section";
+          ).innerHTML = "Jump to section hi";
```

Comment:
Do you still need this else statement here since we know HTML will exist above?

Reply:
Yeah, this is where we set the html in that heading so I think it needs to stay.

Reply:
If you make the update I suggest on line 96 then you don't need to duplicate this.  It would be good to have a single point in the file where the fallback text is defined so that if it's updated, it will be done in one spot instead of having to find all the references.  Also, I suspect you don't want this postfixed with `hi` at the end.